Resources/Private/Fusion/Backend/Root.fusion

```diff
@@ -14,6 +14,7 @@ backend = Neos.Fusion:Template {
         nodeTree = ${Configuration.setting('Neos.Neos.userInterface.navigateComponent.nodeTree')}
         structureTree = ${Configuration.setting('Neos.Neos.userInterface.navigateComponent.structureTree')}
         allowedTargetWorkspaces = ${Neos.Ui.Workspace.getAllowedTargetWorkspaces()}
+        contentCanvas = ${Configuration.setting('Neos.Neos.Ui.configuration.contentCanvas')}
```

Comment:
You could as well expose the whole configuration part, to not have to do it every time. Just need to think a good naming for it ;)

Reply:
What about `uiConfiguration` ?